lib/PicoFeed/Parser/XmlParser.php

```diff
@@ -95,6 +97,18 @@ public static function getHtmlDocument($input)
             $dom->loadHTML($input);
         }
 
+        $this->errors = [];
```

Comment:
this is a static method here so `$this` doesn't exist. I'm fixing it and will push up the changes.